apps/api/src/auth/auth.service.ts

```diff
@@ -273,4 +282,48 @@ export class AuthService {
       },
     )
   }
+
+  async forgotPass(forgotPasswordDto: ForgotPass) {
+    const user = await this.prismaService.person.findFirst({
+      where: { email: forgotPasswordDto.email },
+    })
+    if (!user) {
+      throw new BadRequestException('Invalid email')
+    }
+    const payload = { username: user.email, sub: user.keycloakId }
+    const jtwSecret = process.env.JWT_SECRET_KEY
+    const access_token = this.jwtService.sign(payload, {
+      secret: jtwSecret,
+      expiresIn: '10m',
+    })
+    const appUrl = this.config.get<string>('APP_URL_LOCAL')
+    const link = `${appUrl}/change-password?token=${access_token}`
+    const profile = {
+      email: user.email,
+      firstName: user.firstName,
+      lastName: user.lastName,
+      link: link,
+    }
+    const userEmail = { to: [user.email] }
+    const mail = new ForgotPassDto(profile)
+    await this.sendEmail.sendFromTemplate(mail, userEmail)
+  }
+
+  async recoveryPass(recoveryPasswordDto: RecoveryPasswordDto) {
```

Comment:
this method should be called updateForgottenPassword, because this is what it is used for